install/0000_30_machine-api-operator_01_images.configmap.yaml

```diff
@@ -19,5 +19,6 @@ data:
       "clusterAPIControllerAzure": "registry.svc.ci.openshift.org/openshift:azure-machine-controllers",
       "clusterAPIControllerGCP": "registry.svc.ci.openshift.org/openshift:gcp-machine-controllers",
       "clusterAPIControllerOvirt": "registry.svc.ci.openshift.org/openshift:ovirt-machine-controllers",
+      "clusterAPIControllerKubevirt": "registry.svc.ci.openshift.org/openshift:kubevirt-machine-controllers",
```

Comment:
As I know `registry.svc.ci.openshift.org` is dead now. Shouldn't we use either `registry.ci.openshift.org` or `quay.io` instead?

Reply:
that's a good point, but iirc these all get replaced during the release image process. for this change i reversed the removal i had done, but your comment makes me wonder if we shouldn't update all of these?

cc @joelspeed, wdyt?